services/121-service/src/user/user.service.ts

```diff
@@ -469,4 +471,19 @@ export class UserService {
 
     return userEntity;
   }
+
+  public async getUsers(): Promise<UserEntity[]> {
+    return await this.userRepository.find({
+      select: {
+        id: true,
+        username: true,
+        admin: true,
+        active: true,
+        lastLogin: true,
+      },
+      where: {
+        userType: UserType.aidWorker,
```

Comment:
@Vladimir-Petkov If people register with the Person Affected app they would also appear in the query. That's why I added the where statement to only get aidworker users. (This is hard to know)

Reply:
@RubenGeo Understand, thank you for this feedback. I think only aidWorkers will be in Users table.

Reply:
Unfortunately not .You can try this locally if you want:
- Seed nlrc-multiple
- Go to program 1 and open it for registrations (by going to registration phase)
- Start the pa app (npm run start:pa)
- Open the PA app and register 1 PA
- Check you users tables> select * from users

Reply:
I happens through this endpoint user/person-affected

Reply:
It's very confusing. But it's legacy code from the PA app where people affected used to be able to login and logout